mgmt_cli_test.py

```diff
@@ -1679,52 +1679,77 @@ def create_backup_and_report(self, mgr_cluster, label: str):
         return task
 
     def run_read_stress_and_report(self, label):
-        stress_queue = []
+        stress_queue = {"read": [], "write": []}
 
         for command in self.params.get('stress_read_cmd'):
-            stress_queue.append(self.run_stress_thread(command, round_robin=True, stop_test_on_failure=False))
+            if " write " in command:
+                stress_queue["write"].append(self.run_stress_thread(command))
+            elif " read " in command:
+                stress_queue["read"].append(self.run_stress_thread(command))
+            else:
+                raise InvalidArgument("Unknown stress command")
+
+        def get_stress_averages(queue):
```

Comment:
I'm just wondering if we can benefit from this method somewhere else?
Probably, it makes sense to move it outside the method (`StressLoadOperations`) making it available across the file?
Btw, it can be a case for the whole `run_read_stress_and_report` method.
WDYT?

Reply:
Dunno. It feels like, no one needed it for X years then why to bother and do anything no one will use in another Y years :)